core/state/statedb.go

```diff
@@ -473,6 +473,16 @@ func (s *StateDB) Suicide(addr common.Address) bool {
 	return true
 }
 
+func (s *StateDB) SendAll(addr common.Address) bool {
+	stateObject := s.getStateObject(addr)
+	if stateObject == nil {
+		return false
+	}
+
+	stateObject.sendalled = true
+	return true
```

Comment:
This needs journalling, just like the `Suicide` above does it

Reply:
Ah yes.  good point.